src/bramble/client/main.js

```diff
@@ -920,6 +925,10 @@ define([
         this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_DISABLE_WORD_WRAP"}, callback);
     };
 
+    BrambleProxy.prototype.configureAutoCloseTags = function(options, callback) {
+        this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_CONFIGURE_AUTO_CLOSE_TAGS", args: [ options ]}, callback);
```

Comment:
I see. Are those built in bindings or ones we came up with? If they're "from brackets" then we can keep the code in line with their format.